internal/hcs/errors.go

```diff
@@ -312,6 +312,13 @@ func IsOperationInvalidState(err error) bool {
 	return err == ErrVmcomputeOperationInvalidState
 }
 
+// IsAccessIsDenied returns true when err is caused by
+// `ErrVmcomputeOperationAccessIsDenied`.
+func IsAccessIsDenied(err error) bool {
+	err = getInnerError(err)
+	return err == ErrVmcomputeOperationAccessIsDenied
```

Comment:
nit: Could be 
```go
return ErrVmComputeOperationAccessIsDenied == getInnerError(err)
```

I know this is just following the same for the others however